.nuspec/maui.aotprofile.txt

```diff
@@ -24,9 +24,9 @@ Methods:
 	Android.Content.PM.PackageManager Android.Content.ContextWrapper:get_PackageManager ()
 	Android.Content.Res.AssetManager Android.Content.ContextWrapper:get_Assets ()
 	Android.Content.Res.ColorStateList Android.Content.Res.ColorStateList:ValueOf (Android.Graphics.Color)
-	Android.Content.Res.ColorStateList AndroidX.AppCompat.Widget.TintTypedArray:GetColorStateList (int)
 	Android.Content.Res.ColorStateList Microsoft.Maui.Platform.ColorExtensions:ToDefaultColorStateList (Microsoft.Maui.Graphics.Color)
 	Android.Content.Res.ColorStateList Microsoft.Maui.Platform.ColorStateListExtensions:CreateDefault (int)
+	Android.Content.Res.ColorStateList Microsoft.Maui.PlatformInterop:GetColorStateListForToolbarStyleableAttribute (Android.Content.Context,int,int)
```

Comment:
Looking at your actual changes to the template, they shouldn't affect the AOT profile -- you basically moved files around. Other changes in MAUI, caused these files to change.

Maybe you can just discard any changes to the AOT profiles here? And I'll update them in a separate PR? (I want to time them and understand what changed in here)

Reply:
ah, I think the AOT did not get updated after this was merged: https://github.com/dotnet/maui/pull/4912